src/metabase/util.cljc

```diff
@@ -706,7 +706,7 @@
       [;; strip out any current symbols
        [#"[^\d,.-]+"          ""]
        ;; now strip out any thousands separators
-       [#"(?<=\d)[,.](\d{3})" "$1"]
```

Comment:
What was that lookbehind doing? I have a feeling it had some obscure purpose that an untested feature depends on. Maybe not though.

Reply:
Hi @camsaul, can you check if this lookbehind is needed or not?  Tests still pass when it’s removed.  (I saw it was introduced in https://github.com/metabase/metabase/pull/11716)

Reply:
It does do something, try parsing something like `.001` with and without that lookbehind. Not sure if that's a realistic thing we need to parse or not tho. We should check if that's even something that can come back in a PG `currency` column or not. If it's not then we can go ahead and remove it

Reply:
I’m going to restore the lookbehind since it works fine in latest safari.